data.js

```diff
@@ -286,5 +286,29 @@ var data = [
     'name': 'Brandner',
     'text': ['der politische Gegner als \"Ergebnis von Sodomie und Inzucht\" bezeichnet'],
     'src': ['https://www.noz.de/deutschland-welt/politik/artikel/955751/wie-rechts-ist-die-afd']
+  },
+  {
+    'forename': 'Markus',
+    'name': 'Söder',
+    'text': ['der findet, dass in Klassenzimmer \"Kruzifixe und keine Kopftücher\" gehören'],
+    'src': ['http://www.spiegel.de/fotostrecke/markus-soeder-seine-besten-zitate-fotostrecke-155827-3.html']
+  },
+  {
+    'forename': 'Ursula',
+    'name': 'von der Leyen',
+    'text': ['die schon guttenbergte bevor es in war'],
```

Comment:
I think there's a comma missing after the word "guttenbergte".

Reply:
Oh you're right 🙈

Reply:
Never mind, I'm quite good at finding grammar and/or spelling issues in other people's texts. Also preparing my own PR at the moment, feedback welcome.